core/chains/orm.go

```diff
@@ -40,6 +40,11 @@ type ORM[I, C, N any] interface {
 
 	StoreString(chainID I, key, val string) error
 	Clear(chainID I, key string) error
+
+	// SetupNodes is a shim to help with configuring multiple nodes via ENV.
+	// All existing nodes are dropped, and any missing chains are automatically created.
+	// Then all nodes are inserted, and conflicts are ignored.
+	SetupNodes([]N, []I) error
```

Comment:
It is a matter of taste, but the cryptic names of types are not self-exlpanatory. Maybe if we were adding parameter names, then it will become clear what types to expect..
I mean:
`SetupNodes(nodes []N, ids []I) error`